docs/README.md

```diff
@@ -16,7 +16,253 @@ The Sail-operator does not manage Gateways. You can deploy a gateway manually ei
+
+    Create a self signed root CA and intermediate CAs.
+    ```sh
+    mkdir -p certs
+    pushd certs
+    curl -fsL -o common.mk "https://raw.githubusercontent.com/istio/istio/${ISTIO_VERSION}/tools/certs/common.mk"
+    curl -fsL -o Makefile.selfsigned.mk "https://raw.githubusercontent.com/istio/istio/${ISTIO_VERSION}/tools/certs/Makefile.selfsigned.mk"
+    make -f Makefile.selfsigned.mk root-ca
+    make -f Makefile.selfsigned.mk east-cacerts
+    make -f Makefile.selfsigned.mk west-cacerts
+    popd
+    ```
+
+    Push the intermediate CAs to each cluster.
+    ```sh
+    kubectl --context "${CTX_CLUSTER1}" label namespace istio-system topology.istio.io/network=network1
+    kubectl get secret -n istio-system --context "${CTX_CLUSTER1}" cacerts || kubectl create secret generic cacerts -n istio-system --context "${CTX_CLUSTER1}" \
+      --from-file=certs/east/ca-cert.pem \
+      --from-file=certs/east/ca-key.pem \
+      --from-file=certs/east/root-cert.pem \
+      --from-file=certs/east/cert-chain.pem
+    kubectl --context "${CTX_CLUSTER2}" label namespace istio-system topology.istio.io/network=network2
+    kubectl get secret -n istio-system --context "${CTX_CLUSTER2}" cacerts || kubectl create secret generic cacerts -n istio-system --context "${CTX_CLUSTER2}" \
+      --from-file=certs/west/ca-cert.pem \
+      --from-file=certs/west/ca-key.pem \
+      --from-file=certs/west/root-cert.pem \
+      --from-file=certs/west/cert-chain.pem
+    ```
+
+4. Create Sail CR on east
+
+    ```sh
+    kubectl apply --context "${CTX_CLUSTER1}" -f - <<EOF
+    apiVersion: operator.istio.io/v1alpha1
+    kind: Istio
+    metadata:
+      name: default
+    spec:
+      version: v${ISTIO_VERSION}
+      namespace: istio-system
+      values:
+        pilot:
+          resources:
+            requests:
+              cpu: 100m
+              memory: 1024Mi
+        global:
+          meshID: mesh1
+          multiCluster:
+            clusterName: east
+          network: network1
+    EOF
+    kubectl wait --context "${CTX_CLUSTER1}" --for=jsonpath='{.status.revisions.ready}'=1 istios/default --timeout=3m
+    ```
+
+5. Create east-west gateway on east
+
+    ```sh
+    curl -fsL "https://raw.githubusercontent.com/istio/istio/${ISTIO_VERSION}/samples/multicluster/gen-eastwest-gateway.sh" | \
+      bash -s -- --mesh mesh1 --cluster east --network network1 | \
+      istioctl manifest generate -f - | \
+      kubectl apply --context "${CTX_CLUSTER1}" -f -
+    ```
+
+6. Expose services on east
+
+    ```sh
+    kubectl --context "${CTX_CLUSTER1}" apply -n istio-system -f https://raw.githubusercontent.com/istio/istio/${ISTIO_VERSION}/samples/multicluster/expose-services.yaml
```

Comment:
maybe we can just include the istio `Gateway` resource here? it doesn't seem to change across releases

Reply:
I can include it here. The instructions here are mostly the same as the istio docs.